### ISSUE TITLE
Model does not perform well on real world portrait images/videos

### ISSUE THREAD

**Issue by anilsathyan7** [clovaai/ext_portrait_segmentation#33]
I trained the SINet model on EG1800 and the baidu augmented datasets, using the default settings in the code. I tried both cross entropy and lovasz loss functions for training. Both of them acheived mIOU ~  94.5; but they do not seem to perform well on real world portrait images(even the provided SINet.pth checkpoint). I tried testing the model using the demo [video](https://github.com/dong-x16/PortraitNet/blob/master/myTest/douyu_origin.mp4) from portrait-net repo and 'Visualize_video.py' script. In both cases there were artefacts on the images in the background regions and sometimes they appear on foreground regions also.

Here are the results:-
![outputs_SINet](https://user-images.githubusercontent.com/1130185/95228718-4f279500-081d-11eb-8929-f9861627637a.png)

Output Videos: 
[test_results.zip](https://github.com/clovaai/ext_portrait_segmentation/files/5335237/test_results.zip)

**Comment by dpcq** [comment]
do you try pretrained model?
can you show your Visualize_video code,because i meet some question when i run this code
thx!
